pkg/chartutil/create.go

```diff
@@ -154,7 +154,11 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 const defaultIngress = `{{- if .Values.ingress.enabled -}}
 {{- $fullName := include "<CHARTNAME>.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-apiVersion: networking.k8s.io/v1beta1
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion}}
+apiVersion: networking.k8s.io/v1
+{{- else}}
```

Comment:
Can you make the ending of the else `-}}` so that whitespace is removed

Reply:
Yes, thanks for the comment